templates/sms.1.txt

```diff
@@ -0,0 +1 @@
+{{t "As requested, here is a link to install Firefox on your mobile device: %(link)s"}}
```

Comment:
nit: I wonder if we should bother with submitting this for translation while we still test this EN locales.
I'm not sure what the timeline is to make this available to other locales, maybe something to discuss.